librz/arch/types/types-atomic.sdb.txt

```diff
@@ -185,41 +185,121 @@ type.uint8_t=b
 type.uint8_t.size=8
 type.uint8_t.typeclass=Unsigned Integral
 
+uint_fast8_t=type
+type.uint_fast8_t=b
+type.uint_fast8_t.size=8
+type.uint_fast8_t.typeclass=Unsigned Integral
+
+uint_least8_t=type
+type.uint_least8_t=b
+type.uint_least8_t.size=8
+type.uint_least8_t.typeclass=Unsigned Integral
+
 uint16_t=type
 type.uint16_t=w
 type.uint16_t.size=16
 type.uint16_t.typeclass=Unsigned Integral
 
+uint_fast16_t=type
+type.uint_fast16_t=w
```

Comment:
`w` for `word`? Isn't this again architecture specific if it is a `word` or `half-word`? Also be below with potentiall `w = 32bit`, `d == 64bit`, `q == 128bit`

Reply:
Couldn't find anything in the C standard (looked briefly). But I find `16 = halfword` more intuitive nowadays. The bit-widths don't get smaller in the end.

Reply:
It's how it's defined in the `pf`:
```
Formats:
| b # byte (unsigned)
| B # resolve enum bitfield (see t?)
| c # char (signed byte)
| C # byte in decimal
| d # 0xHEX value (4 bytes) (see 'i' and 'x' formats)
| D # disassemble one opcode
| e # temporarily swap endian
| E # resolve enum name (see t?)
| f # float value (4 bytes)
| F # double float value (8 bytes)
| i # signed integer value (4 bytes) (see 'd' and 'x' formats)
| n # next char specifies size of signed value (1, 2, 4, or 8 byte(s))
| N # next char specifies size of unsigned value (1, 2, 4, or 8 byte(s))
| o # octal value (4 bytes)
| p # pointer reference (2, 4, or 8 bytes)
| q # quadword (8 bytes)
| Q # octoword (uint128_t) (16 bytes)
| r # CPU register (`pf r (eax)plop`)
| s # 32 bit pointer to string (4 bytes)
| s # 32 bit pointer to string (4 bytes)
| t # 32 bit UNIX timestamp (4 bytes)
| T # show ten first bytes of buffer
| u # uleb128 (variable length)
| w # word (2 bytes unsigned short in hex)
```